### DISCUSSION TITLE
storage `expiresIn` param

### DISCUSSION THREAD

**Discussion by mationai** [supabase/supabase#1815] [Questions] [answered]
Is this a bug?

According to https://github.com/supabase/storage-api/blob/master/src/utils/index.ts (and the src repo jsonwebtoken), `expiresIn` accepts string and integer. However, when I pass in '99y', it errors saying "body.expiresIn should be integer"

**Comment by inian** [comment, marked as answer]
Hey @fuzzthink, `signJWT` is an internal function and that happens to be signature of the expiresIn parameter of `jwt.verify` function. For the external API documentation, check out [this page](https://supabase.github.io/storage-api/#/object/post_object_sign__bucketName___)

**Reply by mationai** [reply]
Thank you, and thanks for the link.
